content/docs/started/workstation/getting-started-linux.md

```diff
@@ -19,7 +19,7 @@ GCP, AWS and Azure.
 [snapd](https://snapcraft.io/docs/getting-started), which is pre-installed on Ubuntu
 and Ubuntu derivative operating systems.
 
-1. Install MicroK8s - `snap install microk8s --classic`
+1. Install MicroK8s - `snap install microk8s --classic --channel=1.15/stable`
```

Comment:
According to the [Kubeflow doc](https://www.kubeflow.org/docs/started/k8s/overview/), we should install K8s 1.14. Does channel 1.15 install K8s 1.14? That's confusing!

Reply:
@sarahmaddox Apologies - yes you're correct if we're wanting to ensure compatibility with Kubeflow 0.4 as well. I was focussing on ensure that the latest release (at present 0.7) is supported. Shall I amend?

Reply:
Based on the statement from the docs of "The recommended Kubernetes version is 1.14. Kubeflow has been validated and tested on Kubernetes 1.14." I have updated to specify 1.14 when installing Microk8s. Hopefully should prevent any installation woes!